README.md

```diff
@@ -365,24 +365,26 @@ please create an issue or discussion on our GitHub, https://github.com/UIUC-PPL/
 Charm++ was created and is maintained by the Parallel Programming Lab,
 in the Computer Science department at the University of Illinois at
 Urbana-Champaign.  Our managing professor is Dr. L.V. Kale; students
-and staff have included (in rough time order) Wennie Shu, Kevin Nomura, Wayne
-Fenton, Balkrishna Ramkumar, Vikram Saletore, Amitabh B. Sinha, Manish
-Gupta, Attila Gursoy, Nimish Shah, Sanjeev Krishnan, Jayant DeSouza,
-Parthasarathy Ramachandran, Jeff Wright, Michael Lang, Jackie Wang,
-Fang Hu, Michael Denardo, Joshua Yelon, Narain Jagathesan, Zehra Sura,
-Krishnan Varadarajan, Sameer Paranjpye, Milind Bhandarkar, Robert Brunner,
-Terry Wilmarth, Gengbin Zheng, Orion Lawlor, Celso Mendes, Karthik Mahesh,
-Neelam Saboo, Greg Koenig, Sameer Kumar, Sayantan Chakravorty, Chao Huang,
-Chee Lee, Fillipo Gioachin, Isaac Dooley, Abhinav Bhatele, Aaron Becker,
-Ryan Mokos, Ramprasad Venkataraman, Gagan Gupta, Pritish Jetley, Lukasz
+and staff have included (in rough time order) Wennie Shu, Kevin
+Nomura, Wayne Fenton, Balkrishna Ramkumar, Vikram Saletore, Amitabh
+B. Sinha, Manish Gupta, Attila Gursoy, Nimish Shah, Sanjeev Krishnan,
+Jayant DeSouza, Parthasarathy Ramachandran, Jeff Wright, Michael Lang,
+Jackie Wang, Fang Hu, Michael Denardo, Joshua Yelon, Narain
+Jagathesan, Zehra Sura, Krishnan Varadarajan, Sameer Paranjpye, Milind
+Bhandarkar, Robert Brunner, Terry Wilmarth, Gengbin Zheng, Orion
+Lawlor, Celso Mendes, Karthik Mahesh, Neelam Saboo, Greg Koenig,
+Sameer Kumar, Sayantan Chakravorty, Chao Huang, Chee Lee, Fillipo
+Gioachin, Isaac Dooley, Abhinav Bhatele, Aaron Becker, Ryan Mokos,
+Ramprasad Venkataraman, Gagan Gupta, Pritish Jetley, Lukasz
 Wesolowski, Esteban Meneses, Chao Mei, David Kunzman, Osman Sarood,
 Abhishek Gupta, Yanhua Sun, Ehsan Totoni, Akhil Langer, Cyril Bordage,
 Harshit Dokania, Prateek Jindal, Jonathan Lifflander, Xiang Ni,
 Harshitha Menon, Nikhil Jain, Vipul Harsh, Bilge Acun, Phil Miller,
-Seonmyeong Bak, Karthik Senthil, Juan Galvez, Michael Robson, Raghavendra
-Kanakagiri, and Venkatasubrahmanian Narayanan. Current developers include
-Eric Bohm, Ronak Buch, Eric Mikida, Sam White, Nitin Bhat, Kavitha
-Chandrasekar, Jaemin Choi, Matthias Diener, Evan Ramos, Justin Szaday,
-Zane Fink, and Pathikrit Ghosh.
-
-Copyright (C) 1989-2023 Regents of the University of Illinois
+Seonmyeong Bak, Karthik Senthil, Juan Galvez, Michael Robson,
+Raghavendra Kanakagiri, Venkatasubrahmanian Narayanan. Nitin Bhat, and
+Justin Szaday. Current developers include: Eric Bohm, Ronak Buch, Eric
+Mikida, Sam White, Kavitha Chandrasekar, Jaemin Choi, Matthias Diener,
+Evan Ramos, Zane Fink, Pathikrit Ghosh, Maya Taylor, Aditya Bhosale,
+Mathew Jacob, Tom Vander Aa, and Thomas Quinn.
+
+Copyright (C) 1989-2024 Regents of the University of Illinois
```

Comment:
Isn't it 

```suggestion
Copyright (C) 1989-2024 University of Illinois Board of Trustees
```

?

Reply:
Those suggestions are sound.  However, as this PR was made for the (now defunct) idea of 7.1.0 from that branch, I think we need to make a whole new PR from the 8.0 branch for this.